docs/getting-started/creating-your-first-project.md

```diff
@@ -4,22 +4,13 @@ Now that you have all the dependencies for Apostrophe installed, you can create
 
 ## Generating a test project
 
-Next, install the `apostrophe-cli` tool, which makes bootstrapping a new project a breeze, and create your project.
-
-1. To install the `apostrophe-cli`, run the following command:
-
-    ```bash
-    # Install the apostrophe-cli module
-    npm install apostrophe-cli -g
-    ```
-
-2. Now create a new project:
+1. Create a new project with the Apostrpohe CLI tool:
 
     ```bash
     # Create a project
-    apostrophe create-project test-project
+    apos create-project test-project
     ```
-3. With the project created, you just need to jump in, install the dependencies, and start the server. You'll also be prompted to create a password. Make sure you don't forget it --- you'll need it in a minute.
+2. With the project created, you just need to jump in, install the dependencies, and start the server. You'll also be prompted to create a password. Make sure you don't forget it --- you'll need it in a minute.
```

Comment:
Jump

Reply:
What is it that users actually have to do -- we should be clear.

Reply:
What is "jump?"